pkg/astnormalization/implicit_schema_definition.go

```diff
@@ -0,0 +1,65 @@
+package astnormalization
+
+import (
+	"github.com/jensneuse/graphql-go-tools/pkg/ast"
+	"github.com/jensneuse/graphql-go-tools/pkg/astvisitor"
+)
+
+const (
+	implicitQueryTypeName        = "Query"
```

Comment:
doesn't matter to me, I just thought it would make sense to name it "implicit" because it would make it clear that it's part of the implicit schema definition visitor